examples/README.md

```diff
@@ -0,0 +1,40 @@
+# Examples
+
+Here's a list of examples illustrating how to use the Dfns Typescript sdk / libs, depending on your situation and setup.
+
+### Core SDK
+
+* [service-account](./sdk/service-account): Most basic example of the SDK. Uses a Service Account (aka "machine user").
+* [nextjs-delegated](./sdk/nextjs-delegated): Delegated registration + login + signing with NextJS. This example illustrates a setup where your end-users living in your client app, create their [Delegated Signing](https://docs.dfns.co/dfns-docs/advanced-topics/delegated-signing) wallets (aka "end-user custody").
+* [auth-delegated](./sdk/auth-delegated): Delegated registration + login + signing with ReactJS + Express. Same setup as [nextjs-delegated](./sdk/nextjs-delegated) example, but using different frameworks (express backend + React frontend)
+* [auth-direct](./sdk/auth-direct/): React frontend application directly communicating with Dfns API with users having webauthN credentials
+* [export-wallet](./sdk/export-wallet): Example of how do import wallets from a Nodejs server.
+* [import-wallet](./sdk/import-wallet): Example of how to export wallets from a Nodejs server.
+
+### Libs / Integrations
+
+* Dfns integration with ethersjs v6 (`@dfns/lib-ethersjs6`)
+  * [base-deposit-eth](./libs/ethersjs/v6/base-deposit-eth)
+  * [uniswap](./libs/ethersjs/v6/uniswap)
+* Dfns integration with ethersjs v5  (`@dfns/lib-ethersjs5`)
+  * [arbitrum-deposit-eth](./libs/ethersjs/v5/arbitrum-deposit-eth)
+  * [polygon-deposit-erc20](./libs/ethersjs/v5/polygon-deposit-erc20)
+  * [optimism-deposit-eth](./libs/ethersjs/v5/optimism-deposit-eth)
+  * [biconomy-aa-gasless](./libs/ethersjs/v5/biconomy-aa-gasless)
+* Dfns integration with viem (`@dfns/lib-viem`)
+  * [alchemy-aa-gasless](./libs/viem/alchemy-aa-gasless)
+* Dfns tier-3 integration - Solana (`@dfns/lib-solana`)
```

Comment:
maybe not call these tier3 integraiton. just "Solana web3.js integration". the tier designation doesn't really mean much to the devs. I think just mention the blockchains + the lib names will make what we integrate with more clear.

Reply:
Ok, renamed